src/main/java/org/maxgamer/quickshop/shop/VirtualDisplayItem.java

```diff
@@ -305,6 +305,14 @@ public void spawn() {
             return;
         }
         load();
+
+        // Can't rely on the attachedShop cache to be accurate
+        // So just try it and if it fails, no biggie
+        try {
+            shop.getAttachedShop().updateAttachedShop();
+        } catch (NullPointerException ignored) {
+        }
```

Comment:
Why we can't just avoid NullPointerException like this?
```
Shop attachedshop=shop.getAttachedShop();
if (attachedshop != null) {
attachedshop.updateAttachedShop();
}
```
Also It just in VirtualDisplayItem, what about other display item?

Reply:
I had it like that and it did not work, so I went with this approach. I don't see a problem with it, it solved my issue! In RealDisplayItem the error is not present, and I haven't done anything with ArmorStandDisplayItem since it was deprecated. I also figured now would be a good time to remove ArmorStandDisplayItem since we're redoing the whole item thing kinda.